tracekit.js

```diff
@@ -1093,18 +1092,9 @@ TraceKit.computeStackTrace = (function computeStackTraceWrapper() {
 }());
 
 //Default options:
-if (!TraceKit.remoteFetching) {
-  TraceKit.remoteFetching = true;
-}
-if (!TraceKit.collectWindowErrors) {
-  TraceKit.collectWindowErrors = true;
-}
-if (!TraceKit.linesOfContext || TraceKit.linesOfContext < 1) {
-  // 5 lines before, the offending line, 5 lines after
-  TraceKit.linesOfContext = 11;
-}
-
-
+TraceKit.globalHandling = true;
```

Comment:
Can you please confirm if this will override any default values someone may have had if they did TraceKit.globalHandling = false; before this script was loaded..